presto-main/src/test/java/io/prestosql/operator/scalar/TestDateTimeFunctions.java

```diff
@@ -50,7 +52,7 @@ public void testFormatDateCannotImplicitlyAddTimeZoneToTimestampLiteral()
     public void testLocalTime()
     {
         Session localSession = Session.builder(session)
-                .setStartTime(new DateTime(2017, 3, 1, 14, 30, 0, 0, DATE_TIME_ZONE).getMillis())
+                .setStartTime(Instant.ofEpochMilli(new DateTime(2017, 3, 1, 14, 30, 0, 0, DATE_TIME_ZONE).getMillis()))
```

Comment:
In session builder you can leave an overload taking millis, this would improve readability of the code here.

Reply:
We want to discourage usages where it's constructing the Instant from millis. Ideally, all these calls should use Java's datetime to instant conversions instead of using Joda's DateTime .